synapse/storage/databases/main/registration.py

```diff
@@ -236,13 +236,13 @@ async def get_renewal_token_for_user(self, user_id: str) -> str:
             desc="get_renewal_token_for_user",
         )
 
-    async def get_users_expiring_soon(self) -> List[Dict[str, int]]:
+    async def get_users_expiring_soon(self) -> List[Dict[str, Any]]:
         """Selects users whose account will expire in the [now, now + renew_at] time
         window (see configuration for account_validity for information on what renew_at
         refers to).
 
         Returns:
-            A list of dictionaries mapping user ID to expiration time (in milliseconds).
+            A list of dictionaries, each with a user ID and expiration time (in milliseconds).
```

Comment:
This comment seemed entirely wrong, it becomes a list of dictionaries like `{"user_id": "@foo:matrix.org", "expiration_time": 1234}`.